metadata-ingestion/examples/library/dataset_add_owner_patch.py

```diff
@@ -0,0 +1,24 @@
+from datahub.emitter.mce_builder import make_dataset_urn
+from datahub.emitter.rest_emitter import DataHubRestEmitter
+from datahub.metadata.schema_classes import OwnerClass, OwnershipTypeClass
+from datahub.specific.dataset import DatasetPatchBuilder
+
+# Create DataHub Client
+rest_emitter = DataHubRestEmitter(gms_server="http://localhost:8080")
```

Comment:
we should use DataHubGraph everywhere - it's a strict superset of the functionality in the rest emitter

Reply:
i'm good with this